packages/create-gasket-app/index.js

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+
+const { fork } = require('child_process');
+
+const [,, ...args] = process.argv;
+
+function main() {
+  return fork(path.join(__dirname, 'node_modules', '.bin', 'gasket'), [...['create'], args], {stdio: 'inherit', stdin: 'inherit', stderr: 'inherit'});
```

Comment:
To me this doesn't look like it would work if `npm` hoists `@gasket/cli` to a directory other than something immediately colocated with this file. Given that there are no tests for this, how have you verified that this works?

Reply:
@SivanMehta This is the path to the symlink bin file which npm will point to wherever it was installed/hoisted. You can see the path is not to an actual, i.e. `__dirname/node_modules/@gasket/cli/bin/run` which could change as you mention. So this seems safe.

Reply:
@SivanMehta It's working. That being said, I I should have indicated how I tested this when I submitted the PR.  

On a side note: I did encounter a bug when I ran a quick sanity test and updated PR with a fix.  :sweat_smile:

I tested via the following: 
Test 1). Using npm link. 
Fetch down project, and run `npm link` in project folder. You can now run the following command. **npm create-gasket-app my-app** on your machine. It will validate that create-gasket-app script calls underlying gasket_cli library correctly. One caveat is that the node_modules install fails when creating the gasket app. 

Test 2.) Published package to npm 
I renamed the package name, and published it. Tested npx and yarn scenarios.